docs/Functions.md

```diff
@@ -35,6 +35,11 @@ for (const auto& row : db(select(tab.name, avg(tab.value))
 }
 ```
 
+Use `count(1)` for simply query row count:
+```C++
+unsigned long n = db(select(count(1)).from(tab).unconditionally()).front().count;
```

Comment:
`count` currently represents a signed value, i.e. today it should be 

```c++
const int64_t n = ...
```

It probably would make sense to change [count.h](https://github.com/rbock/sqlpp11/blob/main/include/sqlpp11/aggregate_functions/count.h) from `integral` to `unsigned_integral`.

It would then be correct to say 

```c++
const uint64_t n = ...
```

Reply:
Hi @rbock , I have update the return type to `const int64_t`